### ISSUE TITLE
False positive for NonFullyQualifiedClassName

### ISSUE THREAD

**Issue by gemal** [slevomat/coding-standard#1603]
I have the following php docblock

```
    /**
     * @apiParam (GET) {Number} [page] Page number
     * /

```

Getting:
```
563 | ERROR | [x] Class name \App\Controller\GET in @apiParam
     |       |     should be referenced via a fully qualified
     |       |     name.
     |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName)
```

this happened after updating to 8.13.4

**Comment by kukulich** [comment]
Class names detection in "unknown" annotations is hard. It probably won't be fixed.

**Comment by gemal** [comment]
Would be nice to be able to ignore certain annotations like `@apiParam`

**Comment by gemal** [comment]
perhaps something like:
![image](https://github.com/slevomat/coding-standard/assets/69105/b8f37369-3be2-4c55-9ea4-5b6ef3459dae)

```
	/** @var list<string> */
	public $annotationsIgnore = [];
```

??

**Comment by gemal** [comment]
would you accept a pull request for something like this? I dont want to spend time on looking at it, if you dont accept a fix i this direction

**Comment by kukulich** [comment]
> would you accept a pull request for something like this? I dont want to spend time on looking at it, if you dont accept a fix i this direction

Yes, there's already something similar in other sniff: https://github.com/slevomat/coding-standard/blob/master/doc/namespaces.md#slevomatcodingstandardnamespacesunuseduses-

**Comment by gemal** [comment]
@kukulich can you have a look at the pull ?

**Comment by gemal** [comment]
When is the next scheduled release? Eagerly awaiting a release with my fix so I can update :)
